examples/tutorials/translations/hindi/Part 06 - Federated Learning on MNIST using a CNN.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>This is redundant(english ver).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Again, doesn't show up in my Jupyter notebook  :(</p>

Reply:
done!

Reply:
Then we don't need to worry about it. I got no idea why it's showing up there but as long as branch is clean that's all we want.